### ISSUE TITLE
Channel synchronization failed to import channel: MaxBufferError

### ISSUE THREAD

**Issue by emansom** [Chocobozzz/PeerTube#5226]
### Describe the current behavior

While importing a medium sized channel with ~200+ videos, the import process crashes with a `MaxBufferError`:

### Steps to reproduce

1. Run latest develop branch code.
2. Configure channel synchronization for a medium-sized channel, selecting all videos to be synchronized.
3. See server logs for the error.

### Describe the expected behavior

It not crashing when importing any channel, independent of size.

### Additional information

* PeerTube instance:
  * URL: https://pcbu.nl
  * Version: latest code in develop branch (2022-09-01)
  * NodeJS version: v16.17.0
  * Ffmpeg version: 4.3.4-0+deb11u1

* Browser name, version and platforms on which you could reproduce the bug:
* Link to browser console log if relevant:
* Link to server log if relevant (`journalctl` or `/var/www/peertube/storage/logs/`):

`error[31-8-2022 23:35:56] Failed to import channel De PoppenCast`

```json
{
  "err": {
    "stack": "MaxBufferError: Command failed: /usr/bin/python3 /data/bin/yt-dlp --skip-download --playlist-reverse --dump-json -f bestvideo[vcodec^=avc1][height=720]+bestaudio[ext=m4a]/bestvideo[vcodec!*=av01][vcodec!*=vp9.2][height=720]+bestaudio/bestvideo[vcodec^=avc1][height<=720]+bestaudio[ext=m4a]/bestvideo[vcodec!*=av01][vcodec!*=vp9.2]+bestaudio/best[vcodec!*=av01][vcodec!*=vp9.2]/bestvideo[ext=mp4]+bestaudio[ext=m4a]/best https://www.youtube.com/c/DePoppenCast\nmaxBuffer exceeded ..."
    "//": ".. 90MB+ of spam JSON output"
   }
}
```

**Comment by emansom** [comment]
@Chocobozzz @fflorent Possibly relevant to #5135. Crashes while importing [this channel](https://www.youtube.com/c/DePoppenCast) from YouTube.

The JSON object returned by `yt-dlp` is just too big. Paginated access patterns may be needed.

**Comment by Chocobozzz** [comment]
Thanks, it should be fixed by https://github.com/Chocobozzz/PeerTube/commit/e9fc9e03c120fb048ed00b38157d15144770ec23 (only when using `yt-dlp`)

**Comment by emansom** [comment]
> Thanks, it should be fixed by [e9fc9e0](https://github.com/Chocobozzz/PeerTube/commit/e9fc9e03c120fb048ed00b38157d15144770ec23) (only when using `yt-dlp`)

Confirmed. Functions correctly now. 🎉 👍🏻
